.travis.yml

```diff
@@ -40,6 +51,9 @@ matrix:
       env: TEST_SUITE=backend
 # command to run tests
 script:
+  # We unset GEM_PATH here as a hack to work around Travis CI having
+  # broken all ruby on their guests.  It's possible they fixed their
+  # bug and we can remove this.
```

Comment:
no, they recommended this in a support thread.  My thinking is we should just do a follow-up project of removing it and seeing if things pass in this PR.